### ISSUE TITLE
[Question] Does the loss scaler not work sometimes, leading to model deconvergence?

### ISSUE THREAD

**Issue by ychnh** [NVIDIA/apex#598]
First, thank you so much for developing apex-amp.
This is an amazing tool and I saw improvements of **1/2 memory usage and x4 training speed.**
Please keep up the great work!

I noticed that the loss scaling has been working well in the beginning stages of training but I noticed that around epoch50 there is a loss scaling and the model just starts to deconverge.
Does this happen often or is there something I can do to prevent this from happening?

![image](https://user-images.githubusercontent.com/36415513/68632745-d51a5d80-0532-11ea-984e-73b687f324e4.png)

This is a resnet-unet with ASSP model with pretrained resnet weights on Imagenet.
I did read some [documentation](https://docs.nvidia.com/deeplearning/sdk/mixed-precision-training/index.html) and it seems like ;the loss scaling should handle the issue I had around epoch50.

**Comment by ptrblck** [comment]
Hi @ychnh!
Do you have a baseline to compare the `apex` run against?

What do you mean by "around epoch50 these is a loss scaling"?
Loss scaling should be used in every iteration when training your model with `apex`.
